emsdk_manifest.json

```diff
@@ -425,7 +425,7 @@
     "version": "3.9.2",
     "bitness": 64,
     "arch": "x86_64",
-    "windows_url": "python-3.9.2-1-embed-amd64+pywin32.zip",
+    "windows_url": "python-3.9.2-4-amd64+pywin32.zip",
```

Comment:
This could be a risky change.. so I think it is not a good idea to pull the existing package to a new one. This prevents people from being able to update to it (when they have the old one, emsdk will see it as installed, so it won't attempt to reinstall), but it also prevents people from downgrading if they are met with issues, since the old package will vanish from the registry.

Maybe instead we can register a new python package and leave the old one intact, and then could also take that as an opportunity to update to the latest python 3.10.2?

Reply:
Done